tests/test_hello/GNUmakefile

```diff
@@ -19,6 +19,6 @@
 UKVM_TARGETS=test_hello.ukvm ukvm-bin
 VIRTIO_TARGETS=test_hello.virtio
 MUEN_TARGETS=test_hello.muen
-UKVM_MODULES=
+UKVM_MODULES=gdb # Just check that it builds.
```

Comment:
`ukvm/GNUmakefile` already does the "do all modules build" check, so this is unnecessary.